src/Login/Login.js

```diff
@@ -39,48 +61,39 @@ class Login extends Component {
               onChange={this.updateValue}
             />
           </div>
-          <Link to="/movie" onClick={this.both}>Submit</Link>
+          <button onClick={this.loginHandler}>Submit</button>
+          {this.state.loggedIn && <Redirect to='/movie' />}
```

Comment:
Perhaps instead of the login form understanding that it's logged in and handling a redirect, could a parent component be in charge of rendering different components? Perhaps the Login form could send an action up to a parent to let the parent know that the user is successfully logged in!